getting-started.adoc

```diff
@@ -87,6 +87,24 @@ RHMAP provides a Raincatcher project template to help you run the demonstration.
 
 .. Click *Deploy Cloud App*.
 
+. Upgrade the database into *WFM Demo Project* and *WFM Auth Service*:
+
+.. Select the *Projects* header menu item.
+
+.. Go to the *Cloud Application*
+
+.. Select Data Browser from the left menu
+
+.. Click on *Upgrade Database*
```

Comment:
Click the *Upgrade Database* button in the top right corner, and confirm by clicking *Upgrade Now*. Wait until the upgrade process finishes.